### PR TITLE
<Calendar /> -  fix behavior when focusing on header and pressing arrow down or arrow up

### PR DESCRIPTION
**Pull request by bamai**
https://jira.wixpress.com/browse/DSM-1583

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6846/

File: packages/wix-style-react/src/Calendar/DatePickerHead/DatePickerHead.js [hunk 1]
```diff
@@ -94,8 +94,26 @@ const DatePickerHead = ({
...
+  const _handleKeyDown = event => {
+    // stop arrow up, arrow down, arrow left and arrow right from changing the year and month
+    if (
+      event &&
+      event.keyCode &&
+      (event.keyCode === 38 ||
+        event.keyCode === 40 ||
+        event.keyCode === 37 ||
+        event.keyCode === 39)
+    ) {
```
Comment (sivanharel) [inline packages/wix-style-react/src/Calendar/DatePickerHead/DatePickerHead.js:106]: Let's create a const with the key codes and use it here :)

const KEY_CODES =  {
    arrowUp: 38,
    arrowDown: 40,
    arrowLeft: 37,
    arrowRight: 39,
}

File: packages/wix-style-react/src/Calendar/DatePickerHead/DatePickerHead.js [hunk 2]
```diff
@@ -94,8 +94,26 @@ const DatePickerHead = ({
     );
   };
 
+  const _handleKeyDown = event => {
+    // stop arrow up, arrow down, arrow left and arrow right from changing the year and month
```
Comment (sivanharel) [inline packages/wix-style-react/src/Calendar/DatePickerHead/DatePickerHead.js:98]: Great comment!
I think we could also add that we are preventing the `handleKeyDown` method of the` ReactDayPicker` component to be executed when the focus is on one of the header elements.

**Review by sivanharel — COMMENTED** [review]
Great

File: packages/wix-style-react/src/Calendar/DatePickerHead/DatePickerHead.js [hunk 3]
```diff
@@ -94,8 +94,26 @@ const DatePickerHead = ({
...
+    if (
+      event &&
+      event.keyCode &&
+      (event.keyCode === 38 ||
+        event.keyCode === 40 ||
+        event.keyCode === 37 ||
+        event.keyCode === 39)
```
Comment (EgoziE) [inline packages/wix-style-react/src/Calendar/DatePickerHead/DatePickerHead.js:105]: `keyCode` is deprecated!
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Please use `key` instead
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: packages/wix-style-react/src/Calendar/DatePickerHead/DatePickerHead.js (hunk 3 above)
Comment (EgoziE) [reply packages/wix-style-react/src/Calendar/DatePickerHead/DatePickerHead.js:105]: @sivanharel FYI

Comment (sivanharel) [reply packages/wix-style-react/src/Calendar/DatePickerHead/DatePickerHead.js:105]: Good to know.. The usages of `keyCode` exist a lot in our project so we might need to refactor it.

**Review by sivanharel — APPROVED** [review]
Great:)

**Comment by sivanharel** [comment]
@NeilWix - FYI, @bamai fixed the issue: "when focusing on the arrows or on the month dropdown and pressing the arrow down the year is changed"

**Pull request merged**